template/template_example.Rmd

```diff
@@ -108,32 +116,33 @@ Your new analysis folder should contain:
 
 Your example analysis folder should now look something like this (except with respective experiment accession id and analysis notebook name you are using): 
 
-<img src="screenshots/analysis-folder-structure.png" width=400>
+<img src="https://raw.githubusercontent.com/AlexsLemonade/refinebio-examples/master/template/screenshots/analysis-folder-structure.png" width=400>
 
 In order for our example here to run without a hitch, we need these files to be in these locations so we've constructed a test to check before we get started with the analysis. 
 Run this chunk to double check that your files are in the right place. 
 
 ```{r}
 # Check if the gene expression matrix file is in the data directory stored as `data_dir`
-file.exists(file.path(data_dir, <<DATA_ACCESSION FILENAME>>))
+file.exists(file.path(data_dir, <!--DATA_ACCESSION FILENAME-->))
```

Comment:
I think you want to leave this (and the next one) as it was (pretty sure html comments don't work in code blocks)
```suggestion
file.exists(file.path(data_dir, <<DATA_ACCESSION FILENAME-->))
```

Reply:
I think i missed some of these because I switch strategies.